load-test/scripts/loadTest.js

```diff
@@ -107,6 +109,7 @@ function onSocketConnect(socket, requestStartTime, websocketStartTime, sessionId
       console.log(`EXCEPTION for session id ${sessionId} ` + parsedData.value);
       exceptionCounter.add(1);
     }
+    metricsReporter.onMessage(socket, parsedData);
```

Comment:
I think it would be worth merging this so we have the logic if we want to do this test again. Simplest option might be to add a flag to turn on/off the metrics reporter?

Reply:
Ok cool, I can do that 👍